src/riemann/elastic.clj

```diff
@@ -83,12 +82,17 @@
        (remove streams/expired?)
        (map #(elastic-event % massage))))
 
+(def es-connection (atom nil))
+
 (defn es-connect
   "Connects to the ElasticSearch node.  The optional argument is a url
   for the node, which defaults to `http://localhost:9200`.  This must
   be called before any es-* functions can be used."
   [& argv]
-  (esr/connect! (or (first argv) "http://localhost:9200")))
+  (swap! es-connection
```

Comment:
@jbouwman  `swap!` that doesn't use the args to the fn is a "sign" to use `reset!` instead